src/tests/missions.react.test.js

```diff
@@ -0,0 +1,68 @@
+import {
+  reciveReservationMissions,
+  reciveReservationRockets,
+  addReservationMission,
+  addReservationRocket,
+  removeReservationMission,
+  removeReservationRocket,
+} from '../redux/api/api';
+
+describe('Testing actions of receiving data', () => {
```

Comment:
- [ ] Well done testing the APIs and actions from your app's functionalities 👏 However, please add more unit testing and checking using React Testing Library and Jest. 😄 🙏 

_For example tests for rendering your components, checking elements, and inside functions._